google/resource_compute_subnetwork_test.go

```diff
@@ -184,6 +184,15 @@ func TestAccComputeSubnetwork_flowLogs(t *testing.T) {
 		Providers:    testAccProviders,
 		CheckDestroy: testAccCheckComputeSubnetworkDestroy,
 		Steps: []resource.TestStep{
+			{
+				Config: testAccComputeSubnetwork_flowLogsUnset(cnName, subnetworkName),
```

Comment:
Since flow is not set in this first step, the following steps are updating the resource. We are not testing the case where we want to enable flow at `Create` time. 

I think the best would be to have:
Step 1: flow enabled (test the create path)
Step 2: flow disabled
Step 3: flow enabled (test the update path)

Reply:
You're right, thanks! I'll add another test to follow the Create path.